etc/build/update_homebrew.sh

```diff
@@ -20,6 +21,11 @@ pushd homebrew-tap
     git checkout -b "$BRANCH" || git checkout "$BRANCH"
     VERSION=$version ./update-formula.sh
     git add "pachctl@$MAJOR_MINOR.rb"
+    if [[ $stable ]]; then
+        cp "pachctl@$MAJOR_MINOR.rb" pachctl.rb
```

Comment:
Can't remember, but cp will over write file with out error if file exist right.

Reply:
nah it will create <dest> if it doesn't exist